classes/form/upload_bookings_form.php

```diff
@@ -59,11 +60,14 @@ public function definition() {
         $mform->setType('csvfile', PARAM_INT);
         $mform->addRule('csvfile', get_string('required'), 'required', null, 'client');
 
+        // Allowed statuses minus the '' one, as we have additional help text to explain '' status specifically.
+        $allowedstatuses = array_filter(booking_manager::get_allowed_session_statuses(), fn($status) => !empty($status));
+        $help = nl2br(get_string('facetoface:uploadbookingsfiledesc', 'mod_facetoface', implode(', ', $allowedstatuses)));
```

Comment:
No action here really, just wanted to mark that php code can be so ugly sometimes.

Maybe it's slightly easier on the eyes if one splits the burden. (assignment, then formatting/print)
```php
$help = get_string('this-really-long-thing' ... );
...
$mform->addElement(
  // ...
  '',
  nl2br($help),
);
```
:shrug: